src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5,13 +5,21 @@ databaseChangeLog:
       file: db/changelog/changes/create-table-books.yaml
   - include:
       file: db/changelog/changes/create-table-roles.yaml
+  - include:
+      file: db/changelog/changes/create-table-category.yaml
   - include:
       file: db/changelog/changes/create-users-roles-table.yaml
+  - include:
+      file: db/changelog/changes/create-book-category.yaml
```

Comment:
Let’s  add changesets to the end of the file

Reply:
not fixed